includes/Modules/Ads/Settings.php

```diff
@@ -66,6 +70,10 @@ public function get_owned_keys() {
 			'conversionID',
 			'paxConversionID',
 			'extCustomerID',
+			'customerID',
+			'formattedExtCustomerID',
+			'userID',
+			'accountOverviewURL',
```

Comment:
Are we sure that these fields should be here? Plus, not sure about conversion ID settings mentioned here, it does not make any sense to be in the owned keys list 🤔.

Another question, how `customerID` is different from `extCustomerID`? 🤔

Reply:
As per our Slack conversation, this is sorted @eugene-manuilov .